libr/core/cconfig.c

```diff
@@ -2632,6 +2691,37 @@ R_API int r_core_config_init(RCore *core) {
 	SETPREF ("cfg.wseek", "false", "Seek after write");
 	SETCB ("cfg.bigendian", "false", &cb_bigendian, "Use little (false) or big (true) endianness");
 
+	/* cfg.logging */
+#if DEBUG
+#define R_LOGGING_DEFAULTLEVEL R_LOGLVL_WARN
+#else
+#define R_LOGGING_DEFAULTLEVEL R_LOGLVL_ERROR
+#endif
+	// R2_LOGLEVEL / cfg.logging.level
+	p = r_sys_getenv ("R2_LOGLEVEL");
+	SETICB ("cfg.logging.level", p ? atoi(p) : R_LOGGING_DEFAULTLEVEL, cb_logging_config_level, "Output logging level "\
+	 "(0:SILLY, 1:VERBOSE, 2:DEBUG, 3:INFO, 4:WARN, 5:ERROR, 6:FATAL, 7:FATALTRAP)"
```

Comment:
silly?

Reply:
fataltrap?

Reply:
SILLY was from zf_log I just kept it. FATAL was so a dev could have a fatal error but let it continue on, FATALTRAP being the FATAL that calls r_sys_breakpoint. I guess now that I added in cfg.logging.traplevel it would be possible to make FATAL always breakpoint and a dev/user could just raise the traplevel cfg to stop it from breakpoint-ing.

Would you rather I do that?